charts/renku/values.yaml

```
@@ -613,3 +613,9 @@ graph:
       ## A secret for signing request header tokens to be sent by GitLab with the Push Events
       ## Generated using: `openssl rand -hex 8|base64`
       secret: 1234
+
+  knowledgeGraph:
+    services:
+      renku:
+        ## Renku UI base url e.g. https://dev.renku.ch
+        url: https://dev.renku.ch
```

This value should be blank by default

```suggestion
        url: 
```